packages/docs/src/examples/v-badge/misc-tabs.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-tabs
     dark
```

Comment:
```suggestion
```

Reply:
Dark is redundant here in v3 because colors always have an automatic foreground color now